src/main/java/io/github/pylonmc/pylon/base/BaseItems.java

```diff
@@ -1948,187 +1832,220 @@ private PylonItems() {
-                "TTT",
-                "TXT",
-                "TTT"
-        );
-        recipe.setIngredient('T', Material.TNT);
-        recipe.setIngredient('X', Material.TARGET);
+        PylonItem.register(ExplosiveTarget.Item.class, EXPLOSIVE_TARGET, BaseKeys.EXPLOSIVE_TARGET);
+        BasePages.BUILDING.addItem(BaseKeys.EXPLOSIVE_TARGET);
+
+        ShapedRecipe recipe = new ShapedRecipe(BaseKeys.EXPLOSIVE_TARGET, EXPLOSIVE_TARGET)
+                .shape(
+                        "TTT",
+                        "TXT",
+                        "TTT"
+                )
+                .setIngredient('T', Material.TNT)
+                .setIngredient('X', Material.TARGET);
         recipe.setCategory(CraftingBookCategory.BUILDING);
         RecipeType.VANILLA_SHAPED.addRecipe(recipe);
     }
 
+    public static final ItemStack EXPLOSIVE_TARGET_FIERY = ItemStackBuilder.pylonItem(Material.TARGET, BaseKeys.EXPLOSIVE_TARGET_FIERY)
+            .build();
     static {
-        PylonItem.register(ExplosiveTarget.Item.class, ExplosiveTarget.EXPLOSIVE_TARGET_FIERY_STACK, ExplosiveTarget.EXPLOSIVE_TARGET_FIERY_KEY);
-        GuidePages.BUILDING.addItem(ExplosiveTarget.EXPLOSIVE_TARGET_FIERY_KEY);
-        ShapelessRecipe recipe = new ShapelessRecipe(ExplosiveTarget.EXPLOSIVE_TARGET_FIERY_KEY, ExplosiveTarget.EXPLOSIVE_TARGET_FIERY_STACK);
-        recipe.addIngredient(ExplosiveTarget.EXPLOSIVE_TARGET_STACK);
-        recipe.addIngredient(Material.FIRE_CHARGE);
+        PylonItem.register(ExplosiveTarget.Item.class, EXPLOSIVE_TARGET_FIERY, BaseKeys.EXPLOSIVE_TARGET_FIERY);
+        BasePages.BUILDING.addItem(BaseKeys.EXPLOSIVE_TARGET_FIERY);
+
+        ShapelessRecipe recipe = new ShapelessRecipe(BaseKeys.EXPLOSIVE_TARGET_FIERY, EXPLOSIVE_TARGET_FIERY)
+                .addIngredient(EXPLOSIVE_TARGET)
+                .addIngredient(Material.FIRE_CHARGE);
         recipe.setCategory(CraftingBookCategory.BUILDING);
         RecipeType.VANILLA_SHAPELESS.addRecipe(recipe);
     }
 
+    public static final ItemStack EXPLOSIVE_TARGET_SUPER = ItemStackBuilder.pylonItem(Material.TARGET, BaseKeys.EXPLOSIVE_TARGET_SUPER)
+            .build();
     static {
-        PylonItem.register(ExplosiveTarget.Item.class, ExplosiveTarget.EXPLOSIVE_TARGET_SUPER_STACK, ExplosiveTarget.EXPLOSIVE_TARGET_SUPER_KEY);
-        GuidePages.BUILDING.addItem(ExplosiveTarget.EXPLOSIVE_TARGET_SUPER_KEY);
-        ShapelessRecipe recipe = new ShapelessRecipe(ExplosiveTarget.EXPLOSIVE_TARGET_SUPER_KEY, ExplosiveTarget.EXPLOSIVE_TARGET_SUPER_STACK);
-        recipe.addIngredient(4, ExplosiveTarget.EXPLOSIVE_TARGET_SUPER_STACK);
+        PylonItem.register(ExplosiveTarget.Item.class, EXPLOSIVE_TARGET_SUPER, BaseKeys.EXPLOSIVE_TARGET_SUPER);
+        BasePages.BUILDING.addItem(BaseKeys.EXPLOSIVE_TARGET_SUPER);
+
+        ShapelessRecipe recipe = new ShapelessRecipe(BaseKeys.EXPLOSIVE_TARGET_SUPER, EXPLOSIVE_TARGET_SUPER)
+                .addIngredient(4, EXPLOSIVE_TARGET_SUPER);
```

Comment:
```suggestion
                .addIngredient(4, EXPLOSIVE_TARGET);
```
I think this was typoed before too

Reply:
oh yeah lol